crates/sui-cluster-test/src/faucet.rs

```diff
@@ -101,7 +101,7 @@ impl FaucetClient for LocalFaucetClient {
     async fn request_sui_coins(&self, request_address: SuiAddress) -> FaucetResponse {
         let receipt = self
             .simple_faucet
-            .send(Uuid::new_v4(), request_address, &[50000; 5])
+            .send(Uuid::new_v4(), request_address, &[100000; 5])
```

Comment:
is 50k insufficient in some case?

Reply:
Yes it was. But generally we increased the faucet amount in devnet so it makes sense here too